provider/datagen/src/transform/cldr/cldr_serde/region_displaynames.rs

```diff
@@ -0,0 +1,32 @@
+// This file is part of ICU4X. For terms of use, please see the file
```

Comment:
Nit: It looks like you added `region_displaynames.rs` but didn't delete `territory_displaynames.rs`

Reply:
Done.